deployment/terraform/strings.go

```diff
@@ -99,55 +101,55 @@ http {
 const nginxSiteConfig = `
 upstream backend {
 %s
-        keepalive 64;
+    keepalive 256;
```

Comment:
This is interesting. This is actually not the number of upstream connections per worker, but rather the maximum number of _idle_ keepalive connections that are kept in the worker cache.

From nginx docs:

> It should be particularly noted that the keepalive directive does not limit the total number of connections to upstream servers that an nginx worker process can open. The connections parameter should be set to a number small enough to let upstream servers process new incoming connections as well.

I deliberately set it to 64 after considering these aspects. So it's very interesting to see that bumping this actually has an effect.

Reply:
I think that particularly helped with bursts of requests. `64` was a good value and I've tested with several settings from disabled up to `16384` and the sweet spot seemed to lay somewhere between 200 and 300. We are not talking about huge improvements so it's okay to keep monitoring and possibly iterating. Maybe this will change with the reviewed controller. I can give this a second pass when I am merging that.

Reply:
Yep, so only for a burst of requests, this will be helpful. I guess I was surprised that we _do_ have a "burst" in our load test. Because I was thinking like a smooth ramp up, and then keep the same load.

Totally fine with merging it as is.

Reply:
I am pretty sure it's due to those get users and statuses calls we do in response to ws events.